ownCloud/Viewer/ImageDisplayViewController.swift

```diff
@@ -38,12 +54,59 @@ class ImageDisplayViewController : DisplayViewController {
 		}
 	}
 
+	// MARK: - Frame changes
 	override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
 		super.viewWillTransition(to: size, with: coordinator)
 		self.scrollView?.updateScaleForRotation(size: size)
 	}
+
+	// MARK: - Gesture recognizers actions.
+	@objc func tapToZoom() {
+		guard let scrollView = scrollView else {
+			return
+		}
+
+		if scrollView.zoomScale != scrollView.minimumZoomScale {
+			scrollView.setZoomScale(scrollView.minimumZoomScale, animated: true)
+		} else {
+			scrollView.setZoomScale(scrollView.maximumZoomScale / 3, animated: true)
```

Comment:
@pablocarmu I would avoid using magic numbers in the code

Reply:
you're right, I'll code this in a constant.